docs/sources/reference/run.md

```diff
@@ -656,6 +656,11 @@ this driver.
 Default logging driver for Docker. Writes JSON messages to file. `docker logs`
 command is available only for this logging driver
 
+## Logging driver: syslog
+
+Syslog logging driver for Docker. Writes JSON messages to syslog. `docker logs`
```

Comment:
I initially started off with a JSON logger and then changed it. I'll fix the docs